### DISCUSSION TITLE
Gradle Build failure

### DISCUSSION THREAD

**Discussion by r1shhh1** [oppia/oppia-android#4872] [Q&A (Installation)] [answered]
![Screenshot (55)](https://user-images.githubusercontent.com/74286532/217114655-981afcdf-1713-4fc4-96df-e73d8bdf582d.png)
![Screenshot (57)](https://user-images.githubusercontent.com/74286532/217114658-0eba360a-7ddb-4ac9-bcce-03ca6f205191.png)

I am getting these errors while installing the app for the very first time in my device. I followed all the steps of installation can anyone please help me out?

**Comment by MohitGupta121** [comment]
@r1shhh1 Which Android Studio version you are using? And can you please retry by deleting the existing clone and reclone. I think you missing or added something while building the project 1st time.

**Reply by r1shhh1** [reply]
I am using Android Studio eel version . Will try doing that once again then.

**Reply by MohitGupta121** [reply, marked as answer]
@r1shhh1 yes please do it again and also can you please try using Android Studio Bumblebee Patch 3.

**Reply by r1shhh1** [reply]
Yes Sir! I really appreciate you helping me out here.

**Reply by r1shhh1** [reply]
@MohitGupta121 yeah it works now ! Thanks

**Reply by MohitGupta121** [reply]
@r1shhh1 If your issue resolved then please mark this as answer so that it helpful to others.
If you have any other query feel free to ask in new discussion.
But please mark this as answer first.

**Reply by MohitGupta121** [reply]
@r1shhh1 Still you have any query then please feel free to ask.
But if your this build query is solved then please mark this as answer to help other in the community.
Thanks You.
